helm/ako/crds/ako.vmware.com_aviinfrasettings.yaml

```diff
@@ -103,6 +103,15 @@ spec:
                 type: object
                 required:
                 - shardSize
+              nsxSettings:
+                properties:
+                  project:
+                    type: string
+                  t1lr:
+                    type: string
+                type: object
+                required:
+                - t1lr
```

Comment:
Don't we need to autogenerate code for this change?

Reply:
@abhishekbsingh: This field was already added. But ran that tool. It has created new set of definition for k8 functions as it downloaded new packages. So done this for v1beta1 and v1alpha2